zerver/lib/notifications.py

```diff
@@ -452,6 +453,27 @@ def log_digest_event(msg):
     logging.basicConfig(filename=settings.DIGEST_LOG_PATH, level=logging.INFO)
     logging.info(msg)
 
+def followup_day2_email_delay(user):
+    # type: (UserProfile) -> None
+    days_to_delay = 2
+    user_tz = user.timezone
+    if user_tz == '':
+        user_tz = 'UTC'
+    signup_time = user.date_joined.astimezone(pytz.timezone(user_tz))
```

Comment:
we typically don't use variables that would only appear once. So you can just do
`signup_day = user.date_joined.astimezone(..).isoweekday()`